apps/web/src/pages/Auth/LoginPage/LoginPage.tsx

```diff
@@ -9,6 +9,7 @@ import SEO from "@common/components/SEO/SEO";
 
 import useRoutes from "~/hooks/useRoutes";
 import useBreadcrumbs from "~/hooks/useBreadcrumbs";
+import { useSearchParams } from "react-router-dom";
```

Comment:
a tiny nitpick but, can we move this up with the other external dependency imports? 😅

Reply:
Yeah, for sure.
Lol, I just realized that there was a convention in how these imports were grouped.  :male_detective: